docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -185,9 +199,10 @@ The SQL mapping statement does a number of things:
 
 . Try running some simple queries against the `currency` map. For example, this query returns all data in the map and orders it by the currency code.  
 +
-```bash
-hzc sql "SELECT * FROM currency ORDER BY Code"
-```
+[source,bash]
+----
+clc sql "SELECT * FROM currency ORDER BY Code"
```

Comment:
I think there is a typo in the update? Should be:

`clc sql -c dev "SELECT * FROM currency ORDER BY Code" -f table`

instead of

`clc sql -c dev "SELECT * FROM currencydata ORDER BY Code" -f table`